src/modules/imageresizer/dll/ImageResizerConstants.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include <string>
+
+namespace ImageResizerConstants
+{
+    // Name of the powertoy module.
+    inline const std::wstring ModuleKey = L"Image Resizer";
```

Comment:
should be "ImageResizer" without the space as that was the name that was previously used. This change would make it backward incompatible.

Reply:
The following value seems to be used in the master to register Image Resizer with runner: https://github.com/microsoft/PowerToys/blob/master/src/modules/imageresizer/dll/Resources.resx#L127. Am I missing something?

Reply:
@somil55 the `CSettings` class is used by the shell extension for Image Resizer and it uses it for deciding whether Image Resizer should be shown in the context menu or not, by reading Enable/Disable. It uses the file `image-resizer-settings.json`, and this was added when the setting was moved from registry to json.